dashboard/src/components/LoginForm/OauthLogin.tsx

```diff
@@ -1,29 +1,38 @@
 import { CdsButton } from "@clr/react/button";
+import { useIntl } from "react-intl";
 
 interface ILoginFormProps {
   authenticationError: string | undefined;
   oauthLoginURI: string;
 }
 
 function OAuthLogin(props: ILoginFormProps) {
+  const intl = useIntl();
   return (
     <section className="title" aria-labelledby="login-title" aria-describedby="login-desc">
       <h3 id="login-title" className="welcome">
-        Welcome to <span>Kubeapps</span>
+        {intl.formatMessage({ id: "login-title-welcome", defaultMessage: "Welcome to" })}
+        <span>{intl.formatMessage({ id: "Kubeapps", defaultMessage: "Kubeapps" })}</span>
```

Comment:
I don't think we need this translated in any language do we? Does the product name change?

Reply:
I believe the goal is to include **all** the strings, not only the ones that may require translation.